test/controllers/update/check.your.answers.controller.spec.ts

```diff
@@ -83,6 +87,22 @@ describe("CHECK YOUR ANSWERS controller", () => {
   });
 
   describe("GET tests", () => {
+
+    test(`renders the ${UPDATE_CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
```

Comment:
should this test title mention presenter details? It's same as 106 now.

Reply:
Spot on. Fixed